ConfigWindow.xaml.cs

```diff
@@ -226,8 +239,15 @@ private void GifFps_Click(object sender, RoutedEventArgs e)
         /// <param name="e">Event args</param>
         private void GifResolutionSelector_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            Config.Instance.GifCamWidth = Convert.ToInt32(((string)GifResolutionSelector.SelectedItem).Split('x').First());
-            Config.Instance.GifCamHeight = Convert.ToInt32(((string)GifResolutionSelector.SelectedItem).Split('x').Last());
+            try
+            {
+                Config.Instance.GifCamWidth = Convert.ToInt32(((string)GifResolutionSelector.SelectedItem).Split('x').First());
+                Config.Instance.GifCamHeight = Convert.ToInt32(((string)GifResolutionSelector.SelectedItem).Split('x').Last());
+            }
+            catch (Exception) { }
```

Comment:
Resharper keeps yelling at me if I don't comment these kinds of catches :trogi:
```suggestion
            catch (Exception) { /* ignored */ }
```